src/services/AdminTasks/AdminTaskApi.ts

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios';
+import { Links, Meta } from '../Content/ContentApi';
+
+export interface AdminTaskFilterData {
+  statuses: string[];
+  accountId: string;
+  orgId: string;
+}
+
+// For general use (e.g. introspect)
+export interface JSONTaskPayload {
+  [key: string]: any;
```

Comment:
You may want to read this: https://wanago.io/2022/01/24/index-signatures-typescript/
To attempt to use alternatives to the any type. 
There are cases where it's unavoidable. 
This may not be one of them though.